mappings/net/minecraft/nbt/NbtList.mapping

```diff
@@ -1,4 +1,11 @@
-CLASS net/minecraft/class_2499 net/minecraft/nbt/ListTag
+CLASS net/minecraft/class_2499 net/minecraft/nbt/NbtList
+	COMMENT Represents an NBT list.
+	COMMENT <p>
+	COMMENT Type value: {@code 9}.
```

Comment:
The punctuation here does not match the similar documentation on other classes. All of the 'type value' lines should have consistent punctuation with each other.

Reply:
Only `NbtIntArray` seems to be missing the punctuation, just to be sure, should all `Type value:` lines ends with a comma or not?

Reply:
I would change them all to something like `This class uses the {@linkplain NbtElement#getType NBT type} {@code 9}.`

Reply:
That's very verbose for something as simple as this.

Reply:
I think that it is better to use full sentences in documentation, even if it means adding a few additional words.

Reply:
Erf, in this case it's the type of the element, which I expect to be found and read quickly.

Like "damn what's the type id of NbtByteArray again? ah it's there!" will be much quicker if it's something short and separated.

Reply:
The most important part is the number, which is distinguishable easily from the rest because it's in monospaced font with a different background at the end of the line. Both versions are only a single line.